frontend/src/components/App.tsx

```diff
@@ -21,45 +24,47 @@ interface AppState {
 }
 
 class App extends React.Component<AppProps, AppState> {
-
+  
```

Comment:
Please remove this whitespace (see https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace).